pkg/codegen/testing/test/testdata/simplified-invokes/schema.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
I would love to see a test for enums, objects, archives/assets and resources, both for positional arguments and as return types.

(I would be fine with forbidding returning resources, but that should be an explicit bind error, not a compilation error in generated code.)

Reply:
I will add some tests ✅